sipi/scripts/upload.lua

```diff
@@ -21,8 +22,12 @@ function file_checksum(path)
     return string.match(checksum_orig, "%w*")
 end
 --------------------------------------------------------------------------
+-- Write sidecar file
+function write_sidecar_file()
 
 
+end
```

Comment:
I know almost nothing about lua but this seems like an empty function. Is it correct?

Reply:
Resolved in 5a7f056fb